### DISCUSSION TITLE
Setting grid spacing for ShapeTanimotoDistance

### DISCUSSION THREAD

**Discussion by ljmartin** [rdkit/rdkit#4363] [Q&A] [answered]
Hi all, 
Looking for some help interpreting the usage of ShapeTanimotoDist. I'd like to see the effect of finer grid spacing. Changing the `gridSpacing` parameter apparently doesn't change the results:

```
from rdkit import Chem
from rdkit.Chem.rdShapeHelpers import ShapeTanimotoDist

testmol = Chem.MolFromSmiles('CCCC')
testmolH = Chem.AddHs(testmol)
AllChem.EmbedMultipleConfs(testmolH, 2)

print('change grid spacing:')
for gs in [0.0001, 0.001, 0.01, 0.1, 1]:
    print(ShapeTanimotoDist(testmolH, testmolH, 0,1,gridSpacing=gs))
```
which returns:
```
change grid spacing
0.4034090909090909
0.4034090909090909
0.4034090909090909
0.4034090909090909
0.4034090909090909
```

Am I missing something? Thanks!
Lewis

**Comment by greglandrum** [comment, marked as answer]
Hi @ljmartin,
This is a bug... the `gridSpacing` argument ends up being ignored along the way.
I'll get it fixed.

**Reply by ljmartin** [reply]
fab, thanks! I can do this later, but the reason I was looking into this was to check the precision of the defaults (where large gridspacing should be low precision since it implies lower sampling density, for a given number of points per voxel). Do you have any priors on how precise the default is? i.e. would it be within a couple of decimal points of shape-it?

**Reply by greglandrum** [reply]
We did some investigation of this back when we did the initial implementation, but I know longer have the results.
hmmm.... That'd be a good one for a blog post! :thinking:

**Reply by greglandrum** [reply]
The linked issue is here: https://github.com/rdkit/rdkit/issues/4364
